pkg/namesgenerator/names-generator.go

```diff
@@ -17,7 +17,6 @@ var (
 		"beautiful",
 		"blissful",
 		"bold",
-		"boring",
```

Comment:
Changing to `bored` would be more consistent with the others

Reply:
The vast majority of these terms are neutral or positive. "Bored" still has a slightly negative connotation.

I think there was a lot of intention in choosing names of great scientists and technologists. Suggesting that they're "bored" seems antithetical to that purpose.

Reply:
These adjectives represent states of mind anyone can experience. Many great inventions were done because someone was bored and wanted to do something interesting. Many times smart people are more bored because they have a much higher bar to find something interesting. The point is though, they are human and all humans at some point are bored, angry, cranky, sad, etc. There is nothing wrong with that, I do agree describing someone as boring is not good and doesn't seem to align with the other adjectives.